test/functional/apps/discover/_discover.ts

```diff
@@ -26,14 +27,29 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     defaultIndex: 'logstash-*',
   };
 
+  const logTypes = (msg: string = '') => async () =>
+    log.debug(
+      `\n### Saved Object Types In Index: [.kibana] ${msg}: \n${inspect(
+        await savedObjectInfo.types(),
+        {
+          compact: false,
+          depth: 99,
+          breakLength: 80,
+          sorted: true,
+        }
+      )}`
+    );
+
   describe('discover test', function describeIndexTests() {
     before(async function () {
       log.debug('load kibana index with default index pattern');
 
-      await kibanaServer.savedObjects.clean({ types: ['search'] });
-      await kibanaServer.importExport.load('discover');
-      log.info(
-        `\n### SAVED OBJECT TYPES IN index: [.kibana]: \n\t${await savedObjectInfo.types()}`
+      await esArchiver.load('empty_kibana');
```

Comment:
This should be in the suite that hasn't been migrated yet, rather than in the suite which has been migrated to use importExport and not replace the `.kibana` index at runtime.

Reply:
@spalger the suite before this one is skipped bud.  Which makes this the first test run.  Whatcha think?

Reply:
If this is the first suite run then the .kibana index already exists and is empty and this is unnecessary.

Reply:
Ok, I'll strip it and see what happens.